scripts/evaluation/v0.6-openrouter.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+make install
+
+eval-dev-quality evaluate \
+	--runtime docker \
+	--runtime-image ghcr.io/symflower/eval-dev-quality:197ece8 \
```

Comment:
We should extract the hash into a variable making it more obvious when copying the script for the next run.